odoo/addons/base/models/res_company.py

```diff
@@ -300,3 +300,6 @@ def _get_main_company(self):
             main_company = self.env['res.company'].sudo().search([], limit=1, order="id")
 
         return main_company
+
+    def _get_layout_background(self):
+        return base64.b64encode(open(os.path.join(tools.config['root_path'], 'addons', 'base', 'static', 'img', 'bg_background_template.jpg'), 'rb').read()).decode()
```

Comment:
Unclosed file descriptor + bad usage of `open` (should use `odoo.tools.file_open`)